slurm-mlr/README.md

```diff
@@ -0,0 +1,19 @@
+# Disclaimer
+
+This example shows how to use `drake` to perform a benchmark analysis using two algorithms (KNN, SVM) across two datasets.
+
+The shown tuning ranges and settings of the cross-validation are not meant to make sense. 
+They are chosen so that the example finishes within 5 minutes with the specified resources.
+Runtime will differ depending on how your CPUs are.
+
+For more detailed instructions on how to use `mlr` see https://mlr.mlr-org.com/.
+
+# R interpreter
+
+This example uses [Spack](spack.io) to load R version 3.5.3 in `slurm_clustermq.tmpl`.
+Depending on your HPC setup you might want to replace this line with `module load R` or similar.
+This command is responsible for loading R on the workers.
+
+# Execution
+
+Run `r_make()` to run the whole workflow.
```

Comment:
Maybe for completeness, would it be useful to have a top-level script that calls `r_make()`? Maybe inside a SLURM script so `r_make()` runs on a compute node rather than a login node? See https://github.com/ropensci/drake/issues/812#issuecomment-482938430 and https://github.com/ropensci/drake/issues/812#issuecomment-483621932.

Reply:
Two questions here: 

1. Is this really needed for this example here? I guess this is an edge-case and would make things more complicated for new users. For normal and medium-sized plans `r_make()` from the master node should be fine and the simplest approach. Invoking 'srun` to call `sbatch` with `drake` seems like a real big wrapper to "only" start something..

2. I haven't read the whole issues/conversation but wouldn't that be then limiting resources on the compute node?

> On a related note, I believe it is possible to send the master process to a compute node as well: that is, to use srun to submit a script that calls drake and have drake call sbatch through clustermq.

Reply:
An individual `drake` master process does not need many resources, so it can be a small node with smaller resource requirements than the compute nodes (e.g. for the `clustermq` workers). But as @kendonB explained at https://github.com/ropensci/drake/issues/812#issuecomment-482937298, local resources on the login node could potentially add up. As a general rule, it is better to send any long computations to a compute node rather than a login node.

That said, all we really need for this example is a "run this to run the workflow" script. Login node or not, I think it such a script conceptually valuable for tying everything together. These examples are pedagogical, and newcomers who look at them may not otherwise know where to begin.